### ISSUE TITLE
Polaris Button (and some other polaris components with anchor tags) are unusually big

### ISSUE THREAD

**Issue by Phiiii** [Shopify/polaris-react#626]
I'm having a weird bug with polaris Buttons (and some other components) ever since our team bumped to Polaris 3 and sewing kit 0.64. The polaris `Button` suddenly looks bigger in production, but the weird thing is that _it looks fine in development_ so this might be a `sewing-kit` issue.

One good clue is that buttons with `url` props seems to be the one that swells up. Normal buttons are fine, so the problem might lie in the anchor tags since AFAIK Polaris buttons will output a styled anchor tag if the url prop is provided.

Development:

<img width="323" alt="screen shot 2018-11-15 at 3 06 04 pm" src="https://user-images.githubusercontent.com/43039473/48578572-047a9800-e8e8-11e8-9694-99f61a7f844d.png">

Production:

<img width="413" alt="screen shot 2018-11-15 at 1 27 57 pm" src="https://user-images.githubusercontent.com/43039473/48578545-ec0a7d80-e8e7-11e8-828d-beb83321c716.png">

This doesn't happen with just Buttons. For example the TopBar component have weird padding too, and these are also anchor tags:

Development:

<img width="252" alt="screen shot 2018-11-15 at 2 43 22 pm" src="https://user-images.githubusercontent.com/43039473/48578523-da28da80-e8e7-11e8-9f4b-659d5bac85e9.png">

Production:

<img width="252" alt="screen shot 2018-11-15 at 2 43 06 pm" src="https://user-images.githubusercontent.com/43039473/48578529-df862500-e8e7-11e8-8dae-9763ec2bd63f.png">

- Are you using the React components? (Y/N): Yes
- Polaris version number: 3.0.0
- Browser: Chrome
- Device: Macbook Pro
- Operating System: MacOS

**Comment by dleroux** [comment]
@GoodForOneFare this looks like a sewing kit issue? The Button should render an `anchor` if a URL is included but for some reason it's rendering both the `button` with an `anchor` inside of it. Any idea how we can fix this?

**Comment by dleroux** [comment]
No they are not Andrew. Sorry forgot to add that detail.

**Comment by Phiiii** [comment]
@AndrewMusgrave Nah, I'm using the default polaris Button. I didn't assign any props to `AppProvider` 😁

**Comment by Phiiii** [comment]
The issue is fixed! It's a sewing-kit thing. The fixes is including this file in production in `sewingkit.config.js`:
```
'@shopify/polaris/esnext/styles/global.scss',
```
Or do:

```
plugins.sass({
   autoImportPolaris: true,
});
```

tl;dr the bug is in our own code 😁
